exotica_examples/scripts/example_dynamic_cartpole_simple

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python2
```

Comment:
Python, not Python2

Reply:
This doesn't work with python3 if your bindings are compiled against python2, which is the default, however.

Reply:
`env python` should be set by the user. It works unless the user has a messed up installation. If we keep it at python2 and the user compiled against python3, it breaks.